### ISSUE TITLE
Can I receive the pretrained ghostnet model with width ratio 1.3

### ISSUE THREAD

**Issue by atrah22** [huawei-noah/ghostnet#27]
Hello,

As per the paper, the validation accuracy on imagenet dataset of ghostnet is higher than mobilenet-v3 (75.7%). However, the actual experiment accuracy from this repo is 74% (lower than mobilenet-v3) .
The model ghostnet implemented at this repo is for width_ratio 1.0. Can I receive the pretrained ghostnet model with width ratio 1.3 (ghostnet 1.3x has 75.7% accuracy as per the paper)

**Comment by iamhankai** [comment]
Yes, accuracy of GhostNet-1.0x from this repo is 74%. The pretrained model of GhostNet-1.3x was not provided now.

You can train it by yourself using script in https://github.com/rwightman/pytorch-image-models or https://github.com/megvii-model/ShuffleNet-Series

**Comment by iamhankai** [comment]
Pytorch pretrained model is released: https://github.com/huawei-noah/ghostnet/tree/master/pytorch

**Comment by iamhankai** [comment]
duplicated to https://github.com/huawei-noah/ghostnet/issues/27
